modules/core/04-channel/types/errors.go

```diff
@@ -37,13 +37,14 @@ var (
 	// Perform a no-op on the current Msg
 	ErrNoOpMsg = errorsmod.Register(SubModuleName, 23, "message is redundant, no-op will be performed")
 
-	ErrInvalidChannelVersion   = errorsmod.Register(SubModuleName, 24, "invalid channel version")
-	ErrPacketNotSent           = errorsmod.Register(SubModuleName, 25, "packet has not been sent")
-	ErrInvalidTimeout          = errorsmod.Register(SubModuleName, 26, "invalid packet timeout")
-	ErrInvalidUpgradeTimeout   = errorsmod.Register(SubModuleName, 27, "either timeout height or timeout timestamp must be non-zero")
-	ErrUpgradeSequenceNotFound = errorsmod.Register(SubModuleName, 28, "upgrade sequence not found")
-	ErrUpgradeErrorNotFound    = errorsmod.Register(SubModuleName, 29, "upgrade error receipt not found")
-	ErrInvalidUpgrade          = errorsmod.Register(SubModuleName, 30, "invalid upgrade")
-	ErrInvalidUpgradeSequence  = errorsmod.Register(SubModuleName, 31, "invalid upgrade sequence")
-	ErrUpgradeNotFound         = errorsmod.Register(SubModuleName, 32, "upgrade not found")
+	ErrInvalidChannelVersion           = errorsmod.Register(SubModuleName, 24, "invalid channel version")
+	ErrPacketNotSent                   = errorsmod.Register(SubModuleName, 25, "packet has not been sent")
+	ErrInvalidTimeout                  = errorsmod.Register(SubModuleName, 26, "invalid packet timeout")
+	ErrInvalidUpgradeTimeout           = errorsmod.Register(SubModuleName, 27, "either timeout height or timeout timestamp must be non-zero")
```

Comment:
Unrelated to your changes, but we could maybe remove this error now? Or it could be useful to keep and just change the string to `"invalid upgrade timeout"` as `"invalid packet timeout"` is used above for `ErrInvalidTimeout`

Reply:
opened #3734